portfolio/src/main/webapp/script.js

```diff
@@ -17,13 +17,15 @@
  * Currently just gets a hardcoded message from the server.
  * TODO(psharma): Update this method to use Lambdas.
  */
-function getRandomQuote() {
-  console.log('inside getrandomquote');
-  fetch('/data').then(response => response.json()).then((quote) => {
-    console.log(quote)
-    const quoteContainerElement = document.getElementById('quote-container');
-    quoteContainerElement.appendChild(
-        createListElement('Comment: ' + quote[0]));
+function getComments() {
+  console.log('inside getComments()');
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments)
+    // Build the list of history entries.
+    const historyEl = document.getElementById('history');
+    comments.forEach((line) => {
```

Comment:
call it commentsEl instead of historyEl. Also update the comment to say "Build list of comments"